src/components/AddWalletModal/AddWalletModal.jsx

```diff
@@ -29,8 +30,18 @@ export const AddWalletModal = () => {
     formState: { errors }
   } = useForm();
 
-  const onSubmit = () => {
-    onClose();
+  const onSubmit = (data) => {
+    axios
+      .post('https://exadel-budget-manager.herokuapp.com/api/Wallet', {
```

Comment:
just use Axios instance from API folder, it already has host, so you need an only the path for requests